src/guides/neutral/graverobber.mdx

```diff
@@ -10,15 +10,15 @@ The Graverobber can steal up to one role from a dead player. Before they steal a
 
 ## Role Type
 
-- The Graverobber is seen as member of the village by the Seer.
+- The Graverobber is seen as a Neutral Party by the Seer.
 - The Graverobber is not seen as a user of witchcraft.
 - The Graverobber is seen visiting by the Harlot/Stalker/Familiar.
 - The Graverobber is not seen as a killer by the Adjudicator.
 
 ## Notes
 
 - The information given in the Role Type section is only true if the Graverobber has not taken on a new identity yet - once they have a new role, all role characteristics correspond to that role.
-- Their is no way to distinguish between a player who has a role from the start of the game and a player who has stolen the role during the game.
+- There is no way to distinguish between a player who has a role from the start of the game and a player who has stolen the role during the game.
```

Comment:
This should probably be "There is no appreciable difference" instead of "There is no way to distinguish", since the latter implies that the difference is still somehow important.

Reply:
I think it is though with the possibility of conflicting intel checks from before and after a steal.

Reply:
It's only important for the people that checked them before the steal - once the steal happens, the Graverobber is the new role, and both checks, powers, and victory conditions are the same as if they started with that role. The wording here implies to me that the difference is important for dealing with the Graverobber itself (i.e. there's no way to tell the difference between a Seer by birth and a Seer by Graverobber, but it's Still Somehow Important), and such wording is used correctly in other places (such as Maple Wolf).

Reply:
Didn't feel strongly about this distinction so I left it as-is when merging in, feel free to raise another PR to update it.